Carbon.MassTransit/IServiceCollectionConfiguratorExtensions.cs

```diff
@@ -279,6 +282,27 @@ public static void AddAsyncRequestResponsePatternForResponder<T>(this IServiceCo
             });
         }
 
+        /// <summary>
+        /// ReceiveEndpoint queue will be declared as a quorum queue, if it is already declared as default, it will delete the existing one and create new HA queue
+        /// </summary>
+        /// <param name="cfg"></param>
+        /// <param name="configuration">Configuration</param>
+        public static void AddAsHighAvailableQueue(this IRabbitMqReceiveEndpointConfigurator cfg,
+                                      IConfiguration configuration)
+        {
+            cfg.SetQuorumQueue(3);
```

Comment:
Can quorum count/value comes from settings?

Reply:
Actually, more than 5 replication of quorum is not recommended. As we have mostly 3 or 5 nodes of rabbit, and 3 is also a good way of quoruming, I kept it constant as 3.

1 is not quorum
2 is not recommended as it is not odd number
3 is OK
4 is not recommended as it is not odd number
5 is OK (has more burden comparing it to 3)
5+ not recommended

So, 3  always works best for us, also preventing it to be misconfigured by anyone else from configuration.